nodewatcher/modules/devices/ubnt/unifi.py

```diff
@@ -84,6 +84,15 @@ class UBNTUniFiAPACLite(cgm_devices.DeviceBase):
     }
 
 
+class UBNTUniFiACMesh(UBNTUniFiAPACLite):
+    """
+    UBNT UniFi AC Mesh device descriptor.
+    """
+
+    identifier = 'ub-ac-mesh'
```

Comment:
I don't think this identifier is the best. For UniFi AP AC Lite we have `ub-uap-ac-lite`. So for mesh we would want something like `ub-umesh-ac` as otherwise the UniFi part gets lost. What do you think?